history-server.md

```diff
@@ -0,0 +1,71 @@
+# Spark History Server
+
+### Prerequisites
+
+Required software:
+* K8s cluster
+* [KUDO CLI Plugin](https://kudo.dev/docs/#install-kudo-cli) 0.7.3 or higher
+* [kubens](https://github.com/ahmetb/kubectx)
+
+### Installing Spark Operator with History Server Enabled
+
+```
+kubens spark-operator
+kubectl kudo install ./kudo-operator -p operatorImageName=mesosphere/kudo-spark-operator -p operatorVersion=latest -p enableHistoryServer=true
+```
+
+This will setup Spark History Server Pod and Service with Spark Event Log directory pointing to a location `/opt/spark/work-dir` (which you can change by passing another location to a parameter `historyServerFsLogDirectory`). This location is local to a Pod of Spark History Server. If we wnat to make it persistent and shared between different Pods, we have to create a PVC (Persistent Volume Claim) and pass it while starting the Spark History Server.
+
+Suppose you have created a PVC with name `history-server-pvc`, than you can install Spark Operator using following command by passing PVC name.
+
+```
+kubectl kudo install ./kudo-operator -p operatorImageName=mesosphere/kudo-spark-operator -p operatorVersion=latest -p enableHistoryServer=true -p historyServerPVCName=history-server-pvc
+```
+
+### Creating Spark Application
+
+Make sure `specs/spark-application.yaml` has these two configurations added under `spec.sparkConf`.
+
+```
+"spark.eventLog.enabled": "true"
+"spark.eventLog.dir": "/opt/spark/work-dir"
+```
+
+If PVC is passed while installing the Spark Operator, make sure these two fields are also added with following values:
```

Comment:
How about moving this section to right below the [PVC section](https://github.com/mesosphere/kudo-spark-operator/pull/23/files#diff-3661bdb8914b4bffb2d2878819c5d731R22) so it's a continuation of steps if the user sets the PVC?